storage/src/main/java/org/apache/uniffle/storage/util/StorageType.java

```diff
@@ -20,11 +20,11 @@
 public enum StorageType {
   MEMORY(1),
   LOCALFILE(2),
-  HDFS(4),
-  LOCALFILE_HDFS(6),
+  HADOOP(4),
+  LOCALFILE_HADOOP(6),
```

Comment:
> Could we avoid modifying the config option? Because this is a incompatible change. Or we should make this change compatible. We should make old config option effective.

make sense. Let me rollback that part.